server/route/project.go

```diff
@@ -4,21 +4,112 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	"github.com/go-errors/errors"
+	"github.com/guregu/null"
+
+	"github.com/torinos-io/api/server/middleware"
+	project_service "github.com/torinos-io/api/service/project_service"
+	project_store "github.com/torinos-io/api/store/project_store"
 )
 
 // CreateProject creates project
 func CreateProject(c *gin.Context) {
-	c.JSON(http.StatusOK, gin.H{
-		"message": "Project created",
+	ac := middleware.GetAppContext(c)
+
+	projectStore := project_store.New(ac.MainDB)
+	service := project_service.New(project_service.Context{
+		Config:       ac.Config,
+		ProjectStore: projectStore,
 	})
+
+	cartfileHeader, cartErr := c.FormFile("cartfile_content")
+	podfileHeader, podErr := c.FormFile("podfile_content")
+
+	if cartErr != nil && podErr != nil {
+		c.Error(errors.New("Podfile and Cartfile is empty"))
+		return
+	}
+
+	request := &project_service.UploadRequest{}
+
+	if cartFile, err := cartfileHeader.Open(); err == nil {
```

Comment:
Btw, I usually insert no empty line between `err :=` and `if err`.

I do:

```go
    header, err := c.FormFile("cartfile_content")
    if err != nil {
```

But I don't:

```go
    header, err := c.FormFile("cartfile_content")

    if err != nil {
```